src/commands/copy/copy.ts

```diff
@@ -48,6 +48,21 @@ export const commandCopy = command({
       description: 'Correct content-type from "application/octet-stream" to common formats',
       defaultValueIsSerializable: true,
     }),
+    compress: flag({
+      type: boolean,
+      defaultValue: () => false,
+      long: 'compress',
+      description:
+        'Compress copied files using zstandard (appends `.zst` to target file name when compressed). Note: Will not compress very small files',
+      defaultValueIsSerializable: true,
+    }),
+    deleteSource: flag({
+      type: boolean,
+      defaultValue: () => false,
+      long: 'delete-source',
+      description: 'Delete source files after successful copy or compress action',
+      defaultValueIsSerializable: true,
+    }),
     concurrency: option({ type: number, long: 'concurrency', defaultValue: () => 4 }),
```

Comment:
I wonder if this should be `workers` or `threads` as the actual concurrency is `councurrency * workerConcurrency`

Reply:
Have updated command description/documentation.